sources/north-america/us/pa/PEMA_Orthoimagery_2022.geojson

```diff
@@ -3,14 +3,14 @@
     "properties": {
         "id": "PEMA_Orthoimagery_2022",
         "attribution": {
-            "url": "https://www.pema.pa.gov/911-Program/Partners/NG911-GIS/Aerial-Imagery/Pages/default.aspx",
-            "text": "Pennsylvania Emergency Management Agency",
+            "url": "https://www.pasda.psu.edu/uci/DataSummary.aspx?dataset=5177",
+            "text": "PASDA",
             "required": false
         },
-        "name": "PEMA Orthoimagery - Western (2022)",
-        "icon": "https://www.pema.pa.gov/PublishingImages/logo.svg",
-        "url": "https://imagery.pasda.psu.edu/arcgis/services/pasda/PEMAImagery2022/MapServer/WmsServer?request=getmap&version=1.3.0&service=wms&layers=2&styles=&format=image/jpeg&crs={proj}&width={width}&height={height}&bbox={bbox}",
-        "max_zoom": 18,
+        "name": "PEMA Orthoimagery [2022]",
```

Comment:
I think the previous name would be more accurate. Consistent with this layer’s official title, “PEMA Orthoimagery Western PA”, it covers only a slice of western Pennsylvania (including Pittsburgh). Perhaps the JOSM index could use that name instead?

For reference, a similarly named layer covers the whole state:

https://github.com/osmlab/editor-layer-index/blob/4388011bbba74926d3622a6c6355a05e73ad8e00/sources/north-america/us/pa/PEMA_Orthoimagery.geojson#L10